src/common/fs/write-file-sync.injectable.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import getDirnameOfPathInjectable from "../path/get-dirname.injectable";
+import fsInjectable from "./fs.injectable";
+
+export type WriteFileSync = (filePath: string, contents: string) => void;
+
+const writeFileSyncInjectable = getInjectable({
```

Comment:
Thought: should we permit sync writes and reads at all, as it can cause all kinds of problems (I trust you know)?

We could at least deprecate these new `injectable`s to discourage new use?

Reply:
Yes we have to (for the time being) because all of `getConfigurationFileModel` is `sync` and so are the migrations.

I think it would be good to write our own, but that is for the future.